src/fu-security-attr.c

```diff
@@ -203,6 +203,598 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("System firmware is being completely verified by the "
+					  "processor at system start."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("System firmware is not being completely verified by the "
+					  "processor at system start."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_BOOTGUARD_POLICY) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("Critical failures of system firmware verification cause "
+					  "system startup to stop."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("Critical failures of system firmware verification can "
+					  "be ignored by the user."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_BOOTGUARD_OTP) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("System firmware verification cannot be disabled."));
+		} else {
+			/* TRANSLATORS: longer failure message */
+			g_ptr_array_add(strs, _("System firmware verification has been disabled."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_CET_ENABLED) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The processor could detect and ignore more attack "
+					  "attempts from malicious software."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("The processor cannot detect and ignore more attack "
+					  "attempts from malicious software."));
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_CET_ACTIVE) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The processor will detect and ignore more attack "
+					  "attempts from malicious software."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("The processor will not detect and ignore more attack "
+					  "attempts from malicious software."));
+			check_bios_settings = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_SMAP) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The processor can detect and ignore simple attack "
```

Comment:
what's a simple attack?

Reply:
How about `The processor can detect and ignore malicious attack attempts from malicious software`

Reply:
How about qualifying it `The processor can detect and ignore some attacks from malicious software`

Reply:
WFM, thanks.